static/js/click_handlers.js

```diff
@@ -298,14 +298,6 @@ export function initialize() {
         user_status_ui.update_button();
     });
 
-    // NOTIFICATION CLICK
-
-    $("body").on("click", ".notification", function () {
-        const payload = $(this).data("narrow");
-        ui_util.change_tab_to("#message_feed_container");
-        narrow.activate(payload.raw_operators, payload.opts_notif);
-    });
```

Comment:
Is this not how we handle clicking on a desktop notification taking you to the appropriate view?

Reply:
Ahh, yeah, this is done using a click handler registered in `notifications.js`.  OK.